src/sync/installation.ts

```diff
@@ -63,8 +64,23 @@ export const sortedRepos = (repos: Repositories): [string, RepositoryData][] =>
 			new Date(a[1].repository?.updated_at || 0).getTime()
 	);
 
-const getNextTask = async (subscription: Subscription): Promise<Task | undefined> => {
-	const tasks = taskTypes;
+export const getTargetTasks = async (jiraHost: string, allTasks: TaskType[], targetTasks?: TaskType[]): Promise<TaskType[]> => {
+	if (targetTasks) {
+		return intersection(allTasks, targetTasks);
```

Comment:
originally just returned targetTasks, but if some jerk passed in "cats, commits" as the value we want to filter out the trash ones.